core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentPhase.java

```diff
@@ -309,7 +307,7 @@ public void accept(Path path) {
                                 return;
                             }
                             final String pathName = appClassesDir.relativize(path).toString();
-                            if (!pathName.endsWith(".class") || bytecodeTransformers.isEmpty()) {
```

Comment:
I think the purpose of this check is to avoid doing all the next steps unless there is actually a transformer.  Is there a reason to remove it?

Reply:
Unless I am misreading the code, `bytecodeTransformers` should never be empty at this step, which is why I removed it.

Reply:
Ah, okay.  LGTM then.

Reply:
:+1: